apps/docs/src/app/core/component-docs/text/examples/text-expandable.component.html

```diff
@@ -0,0 +1,8 @@
+<h3>Expandable text</h3>
+<fd-text [text]="text" [maxLines]="3" [expandable]="true"></fd-text>
```

Comment:
There are some differences in Ie11
IE11:
![image](https://user-images.githubusercontent.com/26483208/108351105-5c9a5600-71e5-11eb-94a6-eb7ea3d86905.png)

Chrome:
![image](https://user-images.githubusercontent.com/26483208/108351128-62903700-71e5-11eb-806e-c41f2a9ced4a.png)


I guess it's becuase line-clamp doesn't work on IE11

Reply:
Added fix, there was a calculation issue for the IE part :)